src/lib/file-picker/file-picker.ts

```diff
@@ -29,9 +29,50 @@ declare global {
 }
 
 /**
- * The web component class behind the `<forge-file-picker>` custom element.
- * 
  * @tag forge-file-picker
+ * 
+ * @summary
+ * The file-picker component allows for a user to upload files of their own to the system. The component provides a slot for a 
+ * button, as well as drag-and-drop functionality to launch the system file chooser dialog. There are visual queues to let the 
+ * user know when files they are dragging can be dropped, as well as events that are relayed to the developer to handle files 
+ * that are legal and/or illegal based on the parameters set on the component.
+ * 
+ * The expectation of this component is that it will be used as a familiar element on the page that will let users upload files, 
+ * while providing that visual and functional consistency.
+ *  
+ * @property {string | null} accept - Gets/sets the allowed file types.
+ * @property {string | null} maxSize - Gets/sets the maximum allowed file size.
+ * @property {string | null} capture - Gets/sets the camera to use when capturing video or images.
+ * @property {boolean} multiple - Gets/sets whether multiple files are allowed.
+ * @property {boolean} disabled - Gets/sets whether the file picker is disabled.
+ * @property {boolean} compact - Gets/sets whether the file picker uses the compact variant.
```

Comment:
Do we want to stick with "compact" for the name of this property or transition to something else? I remember there was some discussion about it being inconsistent with similar properties in other components